src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsuserpreferences/controller/CourtsController.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.hmppsuserpreferences.controller
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsuserpreferences.jpa.entity.Users
+import uk.gov.justice.digital.hmpps.hmppsuserpreferences.jpa.repository.UsersRepository
+import java.util.UUID
+
+@RestController
+class CourtsController {
+
+  @Autowired
+  lateinit var repository: UsersRepository
+
+  @ResponseStatus(value = HttpStatus.CREATED)
+  @PutMapping("/users/{uuid}/preferences/courts")
+  fun updatePreferences(@PathVariable uuid: UUID, @RequestBody user: Users): List<String>? =
```

Comment:
Think we should ensure that the response is never a null  but an empty list of strings. If you remove the "?" isn't that saying the response will never be null ?

Reply:
Returns: the saved entities; will never be null. The returned Iterable will have the same size as the Iterable passed as an argument.

Reply:
Sorry, I misunderstood the question. Yes, I'll change it.